src/content/docs/alerts-applied-intelligence/transition-guide.mdx

```diff
@@ -101,7 +101,7 @@ We're slowly rolling out this update to all of our users over the next months.
 
     Alerts (classic) lets you manage events, incidents, and notification channels with the UI you're used to using.
 
-    Use the [events and incidents pages](/docs/alerts-applied-intelligence/new-relic-alerts/alert-incidents/view-violation-event-details-incidents/) to review alerts violations and other events.
+    Use the [events and incidents pages](/docs/alerts-applied-intelligence/new-relic-alerts/alert-incidents/view-violation-event-details-incidents/) to review alerts incidents and other events.
```

Comment:
Alerts Classic does let you review violations, since it's still using the old terminology.